test/percy/spec.jsx

```diff
@@ -133,11 +175,13 @@ Spec.propTypes = {
   propsToList: PropTypes.arrayOf(PropTypes.string),
   omitPropsList: PropTypes.bool,
   backgroundColor: PropTypes.string,
+  testedThemes: PropTypes.arrayOf(PropTypes.string),
 };
 
 Spec.defaultProps = {
   omitPropsList: false,
   listPropsOfNestedChild: false,
+  testedThemes: ['new', 'old'],
```

Comment:
I also considered using `themingContexts={[“global”, “local”]}` prop instead of this API (`testedThemes={["old”, “new”]}`) used in the PR but for me this approach seemed to be less natural from the DX point of view.